features/admin.connections.v1/components/edit/forms/general-details-form.tsx

```diff
@@ -285,6 +286,19 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                     onSubmit={ (values: GeneralDetailsFormValuesInterface): void => {
                         updateConfigurations(values);
                     } }
+                    validate={ (values: GeneralDetailsFormValuesInterface) => {
+                        const errors: Partial<Record<keyof GeneralDetailsFormValuesInterface, string>> = {
+                            image: undefined
+                        };
+
+                        if (values?.image && values.image !== " " && !ImageUtils.isValidImageExtension(values?.image)) {
```

Comment:
`values.image !== " " ` just for clarification, what are we checking in this condition?

Reply:
We may not need that.

```suggestion
                        if (values?.image && !ImageUtils.isValidImageExtension(values?.image)) {
```